xchange-cexio/src/main/java/info/bitrich/xchangestream/cexio/dto/CexioWebSocketOrderBookSubscriptionData.java

```diff
@@ -0,0 +1,27 @@
+package info.bitrich.xchangestream.cexio.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.knowm.xchange.currency.CurrencyPair;
+
+public class CexioWebSocketOrderBookSubscriptionData {
+	@JsonProperty
+	final List<String> pair;
+	
+	@JsonProperty
+	final boolean subscribe;
+	
+	@JsonProperty
+	final int depth;
+
+	public CexioWebSocketOrderBookSubscriptionData(CurrencyPair currencyPair, boolean subscribe) {
+		pair = new ArrayList<String>(Arrays.asList(currencyPair.base.toString(), 
+				currencyPair.counter.toString()));
+		this.subscribe = subscribe;
+		this.depth = 10; //-1 for full depth
```

Comment:
This might be worth exposing at the API layer. Not sure what is the best way to do so because the xchange API does not provide a way of doing this now.